<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideWindow</class>
 <widget class="QMainWindow" name="SideWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Generic</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>27</y>
          <width>84</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera pos:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labCamPos">
        <property name="geometry">
         <rect>
          <x>78</x>
          <y>28</y>
          <width>226</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0.0000, 0.0000, 0.0000</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>55</y>
          <width>265</width>
          <height>291</height>
         </rect>
        </property>
        <property name="title">
         <string>Selected model</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Trick:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtTrickName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtModelName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Materials</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <addaction name="actionLoad_Scene_Graph"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_data_paths"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Scene_Graph">
   <property name="text">
    <string>Load Scene Graph</string>
   </property>
  </action>
  <action name="actionSet_data_paths">
   <property name="text">
    <string>Set data paths</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
